asm_to_pil/src/romgen.rs

```diff
@@ -274,7 +274,7 @@ mod tests {
         let res = generate_rom_str::<Bn254Field>(vm);
 
         assert_eq!(
-            res.get(&parse_absolute_path("VM"))
+            res.get(&parse_absolute_path("::VM"))
```

Comment:
I'm a bit confused by this. If `VM` is not inside a module, what's the `::` of `VM`?

Reply:
In this context, it is a bit redundant (because the function is called `parse_absolute_path`), but I think it is better to be explicit.

What `::VM` means is that it is a symbol called `VM` inside the "top"/"root" or "main" module.